src/docs/product/accounts/pricing.mdx

```diff
@@ -122,15 +122,23 @@ Your volume is the total number of events and attachments — or the total amoun
 
 ### Prepaid (Reserved) Volume {#prepaid-reserved-capacity}
 
-When you buy reserved volume (or quota), you prepay for data at significant discounts. Your prepaid, reserved volume is always applied before on-demand volume kicks in. Unused reserved volume expires at the end of each billing month.
+When you buy reserved volume (or quota), you prepay for data at significant discounts. Your prepaid, reserved volume can be allocated on either a pooled basis between projects, or, if you have [Spend Allocation]{/product/accounts/quotas/spend-allocation/} enabled, it can be set on both a pooled and a per-project basis. Unused reserved volume expires at the end of each billing month.
```

Comment:
```suggestion
When you buy reserved volume (or quota), you prepay for data at significant discounts. Your prepaid, reserved volume can be allocated on either a pooled basis between projects, or, if you have [Spend Allocation](/product/accounts/quotas/spend-allocation/) enabled, it can be set on both a pooled and a per-project basis. Unused reserved volume expires at the end of each billing month.
```